lib/domain/service/notification.dart

```diff
@@ -337,6 +338,41 @@ class NotificationService extends DisposableService {
     }
   }
 
+  /// Clearing all notifications for a specific chat by [chatId].
+  Future<void> clearNotifications(ChatId chatId) async {
```

Comment:
Does this documentation follow Effective Dart: Documentation guidelines?

https://github.com/team113/messenger/pull/1394#discussion_r2324550432